src/components/QueryEditor.tsx

```diff
@@ -142,6 +148,35 @@ export function QueryEditor(props: Props) {
             checked={query.waitForResult}
           />
         </InlineField>
+        <h6>Frames</h6>
+        <InlineField
+          label="Format as"
+          labelWidth={13}
+          tooltip={
+            <>
+              {
+                'Timeseries queries must have times in ascending order, which can be done by adding "ORDER BY <time field> ASC" to the query. '
+              }
+              <a
+                href="https://docs.aws.amazon.com/timestream/latest/developerguide/supported-sql-constructs.SELECT.html"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                See the AWS Docs for more details.
+              </a>
+            </>
+          }
+          interactive
+        >
+          <Select
+            aria-label="Format as"
```

Comment:
```suggestion
        <InlineField
          label="Format as"
          htmlFor="format-as"
          labelWidth={13}
          tooltip={
            <>
              {
                'Timeseries queries must have times in ascending order, which can be done by adding "ORDER BY <time field> ASC" to the query. '
              }
              <a
                href="https://docs.aws.amazon.com/timestream/latest/developerguide/supported-sql-constructs.SELECT.html"
                target="_blank"
                rel="noopener noreferrer"
              >
                See the AWS Docs for more details.
              </a>
            </>
          }
          interactive
        >
          <Select
            inputId="format-as"
```

The `htmlFor` in `<InlineField>` should associate the label with the `<Select>` element with the `inputId`